routes/api/users.js

```diff
@@ -61,4 +61,21 @@ router.put("/:username",(request,response) =>{
      response.send({msg:"User deleted",data:newdata});
 })
 
+router.put("/addTempgrade/:username",(request,response) =>{
+    var tempdata=[];
+    const courseName=request.body.courseName;
+    const grade=request.body.grade;
+    const creditHours=request.body.creditHours
+    for(let i =0;i< users.length;i++){
+        if (users[i].username===request.params.username){
+            tempdata.push({"courseName":courseName, "grade":grade, "creditHours":creditHours})
```

Comment:
in pushing an object you don't have to write the field between quotes.
`teampData.push({coursName: courseName ,....})`